### PR TITLE
doc: makeCacheFromRequest

### PR DESCRIPTION
**Pull request by mGGk-fr**
# What

This MR adds documentation for makeCacheFromRequest

# Preview

https://deploy-preview-781--heuristic-almeida-1a1f35.netlify.app/docs/remixed/api-reference/front-commerce-core/make-cache-from-request

### CODE REVIEW

File: docs/04-api-reference/front-commerce-core/make-cache-from-request.mdx [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+title: "makeCacheFromRequest"
+sidebar_position: 5
+description: "The makeCacheFromRequest allows to cache response for a request."
```
Comment (real34) [inline docs/04-api-reference/front-commerce-core/make-cache-from-request.mdx:4]: **issue:** this is not correct at all. It's a factory for the caching layer.

Several locations of the documentation below is also incorrect.

**suggestion:** @mGGk-fr do you want to proof-read it, or get detailed feebdack on it (in this case it'll likely be faster for me to do another MR fixing this)

**Review by real34 — CHANGES_REQUESTED** [review]

File: docs/04-api-reference/front-commerce-core/make-cache-from-request.mdx (hunk 1 above)
Comment (mGGk-fr) [reply docs/04-api-reference/front-commerce-core/make-cache-from-request.mdx:4]: Added task FC-1754 for follow up
